content/03-reference/01-tools-and-interfaces/02-prisma-client/030-crud.mdx

```diff
@@ -398,6 +400,8 @@ You can also perform nested writes, such as:
 
 The `update` query updates an existing database record. You can use the `select` and `include` options to determine which properties should be included on the returned object.
 
+To perform arithmeic operations (add, subtract, multiply), use [atomic updates](#atomic-operations-on-update) to prevent race conditions.
```

Comment:
```suggestion
To perform arithmeic operations (add, subtract, multiply, divide), use [atomic updates](#atomic-operations-on-update) to prevent race conditions.
```